### DISCUSSION TITLE
Can I join team?

GitHub Username: Aflaungos

### DISCUSSION THREAD

**Discussion by Aflaungos** [ROM-builders/temporary#16726] [Q&A]
https://cirrus-ci.com/task/6551639079780352

Kernel Tree:
https://github.com/Aflaungos/android_kernel_samsung_m23xq

Common Tree:
https://github.com/Aflaungos/android_device_samsung_sm7225-common

Device Tree:
https://github.com/Aflaungos/android_device_samsung_m23xq

Vendor Tree:
https://github.com/Aflaungos/android_vendor_samsung_sm7225-common

**Comment by Apon77** [comment]
please follow #learn instructions in telegram

**Reply by Aflaungos** [reply]
> please follow #learn instructions in telegram

Yes I checked #team note in telegram and saw I wasn't respecting commit history from temporary repo, but I was respecting from my trees and am from temporary too now. Git commands I have knowledge in cherry picking merging pull pushing removing

Can I get added now?
